src/Paramore.Brighter.Extensions.DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -348,7 +348,7 @@ private static IAmAnExternalBusService BuildExternalBus(IServiceProvider service
                 RequestContextFactory(serviceProvider),
                 busConfiguration.OutboxTimeout,
                 busConfiguration.MaxOutStandingMessages,
-                busConfiguration.MaxOutStandingCheckIntervalMilliSeconds,
+                TimeSpan.FromMilliseconds(busConfiguration.MaxOutStandingCheckIntervalMilliSeconds),
```

Comment:
Wouldn't we want the busConfiguration.MaxOutStandingCheckIntervalMilliSeconds to be a TimeSpan, so as to prevent the problem with confusion by the user of the API as to the unit of time. 

So it becomes MaxOutStandingCheckInterval and is a TimeSpan

Reply:
Completely agree, my head was in the external config storage space where as this is internal, will get it fixed